readme.md

```diff
@@ -68,6 +68,7 @@ You might also like [awesome-nodejs](https://github.com/sindresorhus/awesome-nod
 ### CLI
 
 - [zsh-better-npm-completion](https://github.com/lukechilds/zsh-better-npm-completion) - Better ZSH completion for npm.
+- [npmlist](https://github.com/hankchanocd/npmlist) - Fuzzy list npm module's dependencies.
```

Comment:
The description is still not clear and according to your own readme, it works on any package property, not just dependencies.